app/src/org/commcare/dalvik/odk/provider/InstanceProvider.java

```diff
@@ -517,6 +537,10 @@ private void linkToSessionFormRecord(Uri instanceUri) {
             if (FormRecord.STATUS_COMPLETE.equals(current.getStatus())) {
                 try {
                     new FormRecordProcessor(getContext()).process(current);
+                } catch (InvalidStructureException | InvalidStorageStructureException e) {
+                    // record should be wiped when form entry is exited
+                    Logger.log(AndroidLogger.TYPE_ERROR_WORKFLOW, e.getMessage());
```

Comment:
I'm a bit surprised this is the logging category we chose. Are we bubbling these up in some new way?